src/task.ts

```diff
@@ -95,7 +107,11 @@ export default class Task extends EventTarget {
     }
 
     if (this.opts.afterAll != null) {
-      await this.opts.afterAll.call(this);
+      try {
+        await this.opts.afterAll.call(this);
+      } catch (e) {
```

Comment:
is this related to optimizations?

Reply:
no, but because of this there will be an error in the tests